tests/unit/transformers/test_numerical.py

```diff
@@ -1291,61 +1291,58 @@ class TestBayesGMMTransformer(TestCase):
 
     @patch('rdt.transformers.numerical.BayesianGaussianMixture')
     def test__fit(self, mock_bgm):
-        """Test '_fit'."""
+        """Test '_fit'.
```

Comment:
minor: can we switch `'` to `. Or whatever we pick, let's just use throughout the file

Reply:
I missed that one, thanks!